src/Take.Blip.Builder/FlowManager.cs

```diff
@@ -409,7 +410,16 @@ private async Task ProcessActionsAsync(LazyInput lazyInput, IContext context, Ac
                         {
                             actionTrace.ParsedSettings = settings;
                         }
-                        
+
+                        if (action is RedirectAction)
+                        {
+                            var redirect = settings.ToObject<Redirect>(LimeSerializerContainer.Serializer);
+                            if (redirect.Address == "netflix-recomendacao")
+                            {
+                                _logger.Warning($"#lognetflix# ({lazyInput.Message}) - ({redirect.Address}) - ({redirect.Context})");
```

Comment:
This is a public repository, we should never put information about our customers here.

Reply:
And even it was public, we do not do specific things for specific clients.

Reply:
I'll be removed.